src/js/polyfills/webvtt.js

```diff
@@ -655,17 +652,37 @@ BoxPosition.getSimpleBoxPosition = function (obj) {
 
     obj = obj.div ? obj.div.getBoundingClientRect() :
         obj.tagName ? obj.getBoundingClientRect() : obj;
+
+    const trueHeight = obj.height || height;
+
     const ret = {
         left: obj.left,
         right: obj.right,
         top: obj.top || top,
-        height: obj.height || height,
-        bottom: obj.bottom || (top + (obj.height || height)),
+        height: trueHeight,
+        bottom: obj.bottom || (top + trueHeight),
         width: obj.width || width
     };
     return ret;
 };
 
+// Helper function for visualizing box position
+// Helpful when debugging
+function drawBox(b, container, color) { // eslint-disable-line no-unused-vars
```

Comment:
this is not used by the code so would get shipped with the player however the utility of this function in debugging these captions box positions is worth the extra couple kb of delivery imo

Reply:
How do you execute it? Why does it need to be part of the player/this code, vs a snippet in your browser used for testing?

Reply:
fair questions - you would have to call it yourself below. i suppose it's more confusing than anything leaving it in - i just found it extremely helpful. might add as a window global